src/Swup.ts

```diff
@@ -95,6 +95,7 @@ export default class Swup {
 	boundPopStateHandler: (event: PopStateEvent) => void;
 
 	loadPage = loadPage;
+	performPageLoad = performPageLoad;
 	leavePage = leavePage;
 	renderPage = renderPage;
 	replaceContent = replaceContent;
```

Comment:
Up until now, everything was more or less public, and we're introducing private methods. We could specify that it's private so it's reflected in generated types as well, but that will likely cause problems in our methods files too when using these unfortunately. We should figure out how to make private things private for the outside consumers.

Reply:
Well ok, it's actually mostly private and exposed as public, so the problem is there for some time, but still, anything we could do? 🤔

Reply:
True, right now they're kind of half public, half private/undocumented. While we could start adding underscore like `_performPageLoad`, we should do that for all undocumented ones like `renderPage` and `leavePage` as well. Maybe that's a breaking change for v4? Browser support for [private methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) isn't very good at the moment either.

Reply:
Basically, it's a mess now. Let's address later. 😄 
No need for renaming, TypeScript has the support, and as long as it's reflected in types for consumers, I think that's ok.

Reply:
https://github.com/swup/swup/issues/598

Reply:
Perfect, let's brainstorm solutions in the issue.